src/index.html

```diff
@@ -11,6 +11,20 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="big-search-form" method="post" action="#"
+    data-qa="big">
```

Comment:
```suggestion
    <form 
      class="big-search-form" 
      method="post" 
      action="#"
      data-qa="big"
    >
```

check code style everywhere